data/docs/migration/migrate-from-honeycomb/data.mdx

```diff
@@ -0,0 +1,263 @@
+
+### Honeycomb's Data Model
+
+Honeycomb treats most observability data as events with structured fields:
+
+| Data Type | Honeycomb Approach | Characteristics |
+|-----------|-------------------|-----------------|
+| **Traces** | Structured events with trace context | Events with span relationships |
+| **Logs** | Structured events with custom fields | Application logs as events |
+| **Metrics** | Derived columns, custom events, and SLOs | Calculated from events + custom metrics |
+| **Events** | Core data structure | Flexible schema with arbitrary fields |
+
+### SigNoz's Unified Approach
+
+SigNoz provides native OpenTelemetry support with configurable correlation:
+
+| Data Type | SigNoz Approach | Advantages |
+|-----------|----------------|------------|
+| **Traces** | Native OpenTelemetry spans | Standard distributed tracing |
+| **Logs** | Structured logs with trace correlation | Trace-to-log linking when configured |
+| **Metrics** | OpenTelemetry metrics + Prometheus | PromQL querying with trace correlation |
+| **Correlation** | Configurable signal linking with trace context | Enhanced debugging when properly configured |
+
+## Unified Data Flow to SigNoz
+
+### Single OTLP Endpoint Benefits
+
+SigNoz uses a unified OpenTelemetry collector that receives traces, logs, and metrics through a single OTLP endpoint. This seamless approach provides:
+
+- **Automatic correlation**: Traces, logs, and metrics with matching trace context are automatically linked
+- **Unified timestamps**: All signals share consistent timing for accurate debugging
+- **Single configuration**: One endpoint handles all telemetry types without separate setup
+- **Reduced complexity**: No need to configure separate pipelines for different signal types
+
+### How Signals Flow and Correlate
+
+When properly configured, telemetry data flows seamlessly:
+1. Applications send all signals to the same SigNoz OTLP endpoint
+2. SigNoz automatically correlates data using trace context (trace_id, span_id)
+3. Users can jump from metrics to traces to logs with automatic linking
+4. Unified dashboards show all signals together for complete observability
+
+## OpenTelemetry Configuration
+
+Since both Honeycomb and SigNoz use OpenTelemetry, migrating your data involves updating your OTLP endpoint configuration to point to SigNoz instead of Honeycomb. 
+
+**Complete the basic setup first**: Follow the [OpenTelemetry endpoint migration guide](/docs/migration/migrate-from-opentelemetry-to-signoz/) to configure your applications to send data to SigNoz.
+
+This guide covers the data aspects of migration - configuring trace correlation, formatting logs properly, understanding retention differences, and verifying that traces, logs, and metrics are all flowing correctly to SigNoz.
+
+## Traces Migration
```

Comment:
Separate sections for traces and logs are unnecessary as they are covered in migrate from otel section above. Maybe highlight it more